README.md

```diff
@@ -197,3 +207,4 @@ In these exercises, you have a number of impure functions and a number of failin
 * refactor the functions to make the tests pass. It will be useful to open the tests and look at exactly what is expected.
 
 Hint: avoid changing any global variables...
+For help in how to copy an object please see this[example](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/fragments/object-pass-by-reference.js)
```

Comment:
This makes it seem like this element is optional. I think the idea is to cover this concept explicitly in the week at some point. Leaving it up to the students to decide whether or not to read it may lead to the same issue we have now; some people ending up with a vague or incomplete understanding of object identity and pass-by-reference at the end of the week.

I also wonder whether it's at all clear to students by this point in the workshop that they need to clone objects if they've not already been introduced to these ideas. How did your cohort experience this?

I would suggest, at the very least, wording this more strongly to make clear it is not optional. Alternatively, you could stick it between the body of the workshop and the exercises section. Another option is to make it entirely distinct from this workshop and simply schedule some time before starting the workshop to go through and discuss Zooey's example.

I'm happy to go with whichever of those three options will be most effective.

Reply:
I remember that we had quite a bit of time and I just googled with my partner after the mentor told us look up how to copy an object when we got stuck on that bit.  

I've put it in between the body and exercises.  I think with the format of the example, best for them to go through it in their own time.  We can then go around and ask if any issues.